tox_docker.py

```diff
@@ -110,6 +110,8 @@ def getint(reader, key):
             })
         if reader.getstring("ports"):
             image_configs[image]["ports"] = reader.getlist("ports")
+        if reader.getstring("links"):
+            image_configs[image]["links"] = reader.getlist("links")
```

Comment:
It's probably better to validate the format (and values, if possible?) of the links here, in terms of which phase of the tox run gets the failure.

Reply:
Agreed - will see what I can work out

Reply:
validation requires access to the docker containers - do you care about validating twice? could partially validate upfront and then let it run again without too much trouble

Reply:
Yeah, if we don't know the names of all the other images, we can skip that part, but at least the format validation can happen here.